docs/coding.rst

```diff
@@ -457,15 +457,15 @@ If you'd rather have a fully separate template for your test, you can use the
 
     # urls.py
 
-    from django.conf.urls import url
+    from django.urls import path
 
     from bedrock.utils.views import VariationTemplateView
 
     urlpatterns = [
-        url(r'^testing/$',
-            VariationTemplateView.as_view(template_name='testing.html',
-                                          template_name_variations=['1', '2']),
-            name='testing'),
+        path(r"^testing/$",
```

Comment:
Same here - `re_path` seems more appropriate - or we change `^testing/$` to just `testing/`